guide/packages.md

```diff
@@ -152,6 +152,16 @@ The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [
 | `membrane_server_icecast`      | [Suspended]                                                                                                                         | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-server-icecast)                                                                                                                                                                                                                                |
 | `membrane_telemetry_dashboard` | Introduction to integrating `membrane_timescaledb_reporter` repository with `membrane_dashboard` to monitor your pipeline behaviour | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_telemetry_dashboard)                                                                                                                                                                                                                           |
 
+## WebRTC clients
+| Package                        | Description                                                                                                                         | Links                                                                                                                                                                                                                                                                                                                                                                                |
+| ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `membrane-webrtc-js` | JS/TS client library for Membrane RTC Engine | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-webrtc-js/) [![npm](https://api.iconify.design/akar-icons:npm-fill.svg?height=20)](https://www.npmjs.com/package/@membraneframework/membrane-webrtc-js)                                                                                                                                                                                                                           |
```

Comment:
This icon is not really visible for me, I'd use a badge more consistent with the rest of badges here - the one used in readme: https://github.com/membraneframework/membrane-webrtc-js/blob/master/README.md

Reply:
react-native-membrane-webrtc is also published on npm so we could add badge for it too